README.md

```diff
@@ -1,19 +1,192 @@
+    - Equality semantics: (#xxx)
+        - `===` and SameValueZero compare the normalized values
+        - `==`, `<`, etc can compare BigDecimals to other numeric types, according to their mathematical values (following BigInt)
+        - SameValue/Object.is compares according to the representation of BigDecimal (just one NaN, but differentiating trailing zeroes)
+- Library features:
+    - The `BigDecimal` constructor can be used to convert from other numerical types or strings. `Number` and `BigInt` can cast from `BigDecimal` as well.
+    - BigDecimal.prototype.toString() removes trailing zeros/normalizes by default (??? #xxx)
+    - toExponential, toFixed, toPrecision
+    - Additional library functions: (#xxx)
+      - quantum? compareTotal?
+      - partition? divmod? div?
+      - round? sqrt? trig fns?
+  - Intl.NumberFormat.prototype.format transparently supports BigDecimal (#xxx)
+  - BigDecimal64Array and BigDecimal128Array (binary format implementation-defined to be one of the two IEEE formats, and then dataview methods take flag; #xxx)
+
+## Open questions
+
+This list is very incomplete. Everything should be considered an open question at this point. A few questions which we'd especially like feedback:
+- How should toString() interact with trailing zeros? [#xxx]()
+- How (if at all) should we represent rounding modes? [#xxx]()
+- How should operations which necessarily round, e.g., division, be supported? [#xxx]()
+- What standard library functions should exist? [#xxx]()
+
+## History and related work
+
+The need for accurately representing decimal quantities is not new or unique to our current circumstances. That's why there are a number of popular JS ecosystem libraries for decimal, why many other programming languages, databases and standards have built-in data types for this purpose, and why TC39 has been considering adding Decimal for at least 12 years.
+
+### Related JS ecosystem libraries
+
+JavaScript programmers are using decimal data types today with various ecosystem libraries. The most popular three on npm are each by [MikeMcl](https://github.com/mikemcl):
+- [decimal.js](http://mikemcl.github.io/decimal.js/)
+- [big.js](https://mikemcl.github.io/big.js/)
+- [bignumber.js](https://mikemcl.github.io/bignumber.js/)
+
+These packages have some [interesting differences](https://github.com/MikeMcl/big.js/wiki), but there are also many similarities:
+- APIs are based on JavaScript objects and method calls
+- Rounding modes and precision limits are settings in the constructor
+- Inherently rounding operations like sqrt, exponentiation, division are supported
+
+The initial proposal in this document suggests some differences, described above.
+
+We plan to investigate the experiences and feedback developers have had with these and other existing JavaScript librariesso that we can learn from them in the design of BigDecimal. The discussion continues in #xxx.
```

Comment:
```suggestion
We plan to investigate the experiences and feedback developers have had with these and other existing JavaScript libraries so that we can learn from them in the design of BigDecimal. The discussion continues in #xxx.
```